src/screens/LoginPage/LoginPage.tsx

```diff
@@ -53,17 +54,14 @@ function LoginPage(): JSX.Element {
                 }}
                 required
               />
-              <input
-                type="submit"
-                className="btn"
-                value="Login"
-                onClick={login_link}
-              />
+              <button type="submit" className={styles.loginbtn} value="Login">
+                Login
+              </button>
             </form>
           </div>
         </section>
       </section>
-    </>
+    </div>
```

Comment:
don't close it in whole </div> either move the </> block below div

Reply:
okay

Reply:
who is not recommending it 

i just checked on the official website  of react it is there

Reply:
https://reactjs.org/docs/fragments.html#short-syntax

Reply:
it is the newer way to write the fragment